run-uts

```diff
@@ -12,7 +12,7 @@ fi
 # Run tests in random order find tests recursively (-r).
 echo WHAT: $WHAT
 echo SKIP: $SKIP
-ginkgo -cover -r --skipPackage $SKIP $WHAT
+ginkgo -cover -r --skipPackage $SKIP $WHAT || { sleep 5; exit 1; }
```

Comment:
Bash on Windows + Docker on Windows didn't produce any output at compilation error, possibly container was killed to soon to flush stderr or something like that. It was super confusing just to see blank screeen, fan spinning and then back to shell prompt with no indication on what went wrong. At some point I added sleep and it solved it